packages/logging/src/log.js

```diff
@@ -136,6 +137,27 @@ Log.formatName_ = function(projectId, name) {
   return path + name;
 };
 
+Log.prototype.assignDefaultMetadata_ = function(entry, callback) {
+  var entryMetadata = entry.metadata;
```

Comment:
This function will be updated, but given the path from Log.entry this function will not modify any object from user-space - it will instead modify the object being generate in this function with properties from the user-space object overwriting via extend.